autoload/neoformat.vim

```diff
@@ -250,6 +251,9 @@ function! s:generate_cmd(definition, filetype) abort
     let _fullcmd = join(inline_environment, ' ') . ' ' . executable . ' ' . join(args_expanded) . ' ' . (no_append ? '' : path)
     " make sure there aren't any double spaces in the cmd
     let fullcmd = join(split(_fullcmd))
+    if !using_stderr
+        let fullcmd = fullcmd . ' 2>> ' . expand(tmp_dir . '/stderr.log')
```

Comment:
What were you attempted to do with this expand call here? It doesn't contain any keywords, and in that case it returns an empty string. So it seems that this code is broken here, as it tries to run a command with 2>> tacked on the end, with no file to redirect to.

Reply:
Could you provide such path for which `expand` would return empty string? I see no mention of that in documentation (`:h expand()`). It's used here to make sure that special signs such as `~` are expanded, although, as I now see, `tmp_dir`is expanded at the definition, so it could be removed. Still, I believe it isn't breaking anything, it's just redundant.

Reply:
I can't sure out what is going on with the behavior of expand(). For example:

  echo expand("/tmp") -> /tmp
  echo expand ("/tmp/neoformat") -> ''

Both directories exist. Strange. Anyway it sounds like fnamemodify() is what should be used in this case? But as you say, it's redundant, so maybe it should just be removed.

Reply:
Obviously there is something wrong with your (n)vim installation. Both these examples expand correctly on my machine. And path doesn't have to exist on the filesystem. And `fnamemodify` is not the function that should be used in this case.